Runtime/Scripts/LocalFileProvider.cs

```diff
@@ -0,0 +1,159 @@
+// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
I used and modified an idea from here to fix our scenario https://github.com/atteneder/glTFast/issues/133